pkg/config/config_test.go

```diff
@@ -260,3 +262,47 @@ func Test_MergeInfraParams(t *testing.T) {
 		})
 	}
 }
+
+func TestReadConfigReader(t *testing.T) {
+	tests := []struct {
+		name string
+		cfg  Application
+	}{
+		{
+			name: "empty config",
+			cfg:  Application{},
+		},
+		{
+			name: "config with imports",
+			cfg: Application{
+				Imports: map[core.ResourceId]string{
+					{Provider: "prov", Type: "type", Namespace: "ns", Name: "name"}: "1",
+					{Provider: "prov", Type: "type", Name: "name"}:                  "2",
+				},
+			},
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			for _, cfgFmt := range []string{"json", "yaml", "toml"} {
+				t.Run(cfgFmt, func(t *testing.T) {
+					assert := assert.New(t)
+					tt.cfg.Format = cfgFmt
+
+					buf := new(bytes.Buffer)
+					err := tt.cfg.WriteTo(buf)
+					if !assert.NoError(err) {
+						return
+					}
+					t.Logf("config:\n%s", buf.String())
```

Comment:
TIL this call!

Will this cause output to stderr during `go build ./...`, and if so, do we still need it (or was it just useful during debugging)? Just to make sure we don't start adding noise that makes us ignore potentially useful warnings/etc in the future.

Reply:
go build doesn't run tests. If you mean `go test` it will print if `-v` is specified or the test fails.

Reply:
Ah, yes I did mean `go test` — and only printing on `-v` is perfect. Thanks!